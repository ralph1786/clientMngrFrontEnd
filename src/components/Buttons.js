import React from "react";
import { Link } from "react-router-dom";
import "./Button.css";

const Buttons = () => {
  return (
    <div className="button">
      <Link to="/login">
        <button className="btn-hover color-1">Provider</button>
      </Link>
      <Link to="/">
        <button className="btn-hover color-1">Parent</button>
      </Link>
    </div>
  );
};

export default Buttons;
