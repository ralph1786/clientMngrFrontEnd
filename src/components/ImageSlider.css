/* .image-slider {
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  grid-column: 1/13;
} */

.carousel.carousel-slider {
  height: 550px;
}

.carousel .slide img {
  height: 550px;
}

.carousel .thumbs {
  margin-left: 30%;
}
