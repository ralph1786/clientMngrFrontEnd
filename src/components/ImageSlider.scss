.carousel .carousel-slider {
  height: 700px;
}

.carousel .slide img {
  margin-top: 4%;
  height: 700px;
}

.carousel .thumbs {
  margin-left: 33%;
}

@media (max-width: 1024px) {
  .carousel {
    .thumbs {
      margin-left: 24%;
    }
    .slide {
      img {
        margin-top: 6%;
      }
    }
  }
}

@media (max-width: 768px) {
  .carousel {
    .carousel-slider {
      height: 450px;
    }
    .slide {
      img {
        height: 450px;
        margin-top: 9%;
      }
    }
    .thumbs {
      margin-left: 17%;
    }
  }
}

@media (max-width: 414px) {
  .carousel {
    .carousel-slider {
      height: 250px;
    }
    .slide {
      img {
        height: 250px;
        margin-top: 12%;
      }
    }
    .thumbs {
      margin-left: -10px;
      li {
        width: 60px;
      }
    }
  }
}

@media (max-width: 375px) {
  .carousel {
    .carousel-slider {
      height: 225px;
    }
    .slide {
      img {
        height: 225px;
        margin-top: 13%;
      }
    }
    .thumbs {
      margin-left: -32px;
    }
  }
}

@media (max-width: 360px) {
  .carousel {
    .carousel-slider {
      height: 200px;
    }
    .slide {
      img {
        height: 200px;
        margin-top: 14%;
      }
    }
    .thumbs {
      margin-left: -19px;
      li {
        width: 50px;
      }
    }
  }
}
