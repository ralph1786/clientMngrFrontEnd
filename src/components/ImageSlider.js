import React from "react";
import "./ImageSlider.css";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import { Carousel } from "react-responsive-carousel";

const ImageSlider = () => {
  return (
    <Carousel infiniteLoop={true} autoPlay={true}>
      <div>
        <img
          src="https://cdn.pixabay.com/photo/2016/06/09/02/48/crayons-1445057_1280.jpg"
          alt="slider"
        />
      </div>
      <div>
        <img
          src="https://cdn.pixabay.com/photo/2017/08/21/19/09/child-2666462_1280.jpg"
          alt="slider"
        />
      </div>
      <div>
        <img
          src="https://cdn.pixabay.com/photo/2017/05/07/22/37/kids-2293835_1280.jpg"
          alt="slider"
        />
      </div>
      <div>
        <img
          src="https://cdn.pixabay.com/photo/2018/09/13/22/47/toys-3675934_1280.jpg"
          alt="slider"
        />
      </div>
      <div>
        <img
          src="https://cdn.pixabay.com/photo/2016/11/28/10/48/child-1864718_1280.jpg"
          alt="slider"
        />
      </div>
    </Carousel>
  );
};

export default ImageSlider;
