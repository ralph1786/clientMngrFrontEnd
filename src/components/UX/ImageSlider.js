import React from "react";
import "./ImageSlider.scss";
import "react-responsive-carousel/lib/styles/carousel.min.css";
import { Carousel } from "react-responsive-carousel";
import image1 from "../../assets/images/image1.jpg";
import image2 from "../../assets/images/image2.jpg";
import image3 from "../../assets/images/image3.jpg";
import image4 from "../../assets/images/image4.jpg";
import image5 from "../../assets/images/image5.jpg";

const ImageSlider = () => {
  return (
    <Carousel infiniteLoop={true} autoPlay={true}>
      <div>
        <img src={`${image1}`} alt="slider" />
      </div>
      <div>
        <img src={`${image2}`} alt="slider" />
      </div>
      <div>
        <img src={`${image3}`} alt="slider" />
      </div>
      <div>
        <img src={`${image4}`} alt="slider" />
      </div>
      <div>
        <img src={`${image5}`} alt="slider" />
      </div>
    </Carousel>
  );
};

export default ImageSlider;
