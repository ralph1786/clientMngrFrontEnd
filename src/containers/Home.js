import React, { Component } from "react";
import ImageSlider from "../components/ImageSlider";
import Buttons from "../components/Buttons";

class Home extends Component {
  render() {
    return (
      <div>
        <ImageSlider />
        <Buttons />
      </div>
    );
  }
}

export default Home;
