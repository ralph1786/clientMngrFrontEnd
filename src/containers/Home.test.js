import React from "react";
import { shallow } from "enzyme";
import Home from "./Home";
import Buttons from "../components/Buttons";
import ImageSlider from "../components/ImageSlider";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Home />);
});

it("creates a snapshot of the Home component", () => {
  expect(wrapper).toMatchSnapshot();
});

it("checks that the ImageSlider component is rendered", () => {
  expect(wrapper.find(ImageSlider)).toBeTruthy();
});

it("checks that the Buttons component is rendered", () => {
  expect(wrapper.find(Buttons)).toBeTruthy();
});
