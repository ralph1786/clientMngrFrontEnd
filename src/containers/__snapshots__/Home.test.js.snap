// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a snapshot of the Home component 1`] = `
<Fragment>
  <ImageSlider />
  <Buttons />
</Fragment>
`;
